packages/core/src/index.ts

```diff
@@ -93,7 +93,7 @@ export {
   deleteCookie,
   resetInitCookies,
 } from './browser/cookie'
-export { CookieStore, WeakRef, WeakRefConstructor } from './browser/types'
+export { CookieStore, WeakRef, WeakRefConstructor } from './browser/browser.types'
```

Comment:
💬 Suggestion: this looks a bit redundant and less helpful. Maybe we could go with the official TS file naming, such as, `dom.custom.types` or other more specific ones?